specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/loadtestservice.json

```
@@ -0,0 +1,2482 @@
+          }
+        },
+        "percentiles": {
+          "description": "List of percentiles values for response time, supported values 50,90,99,95. Default value is 50th percentile.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "groupByInterval": {
+          "description": "For test duration less than 10 minutes group by time interval can be any one of 5s,10s,1m,5m.\\n\\nFor test duration greater than 10 minutes, group by time interval can be any one of 1m,5m,1h. Default value is 1m.",
+          "type": "string"
+        },
+        "startTime": {
+          "format": "date-time",
+          "description": "Start time",
+          "type": "string"
+        },
+        "endTime": {
+          "format": "date-time",
+          "description": "End time",
+          "type": "string"
+        }
+      }
+    },
+    "ClientMetricsResults": {
+      "type": "object",
+      "properties": {
+        "testRunId": {
+          "description": "Test run name for which client metrics results is required.",
+          "type": "string"
+        },
+        "timeSeries": {
+          "$ref": "#/definitions/Series"
+        }
+      }
+    },
+    "DefaultServerMetricsConfigListModel": {
+      "description": "Default server metrics config",
+      "type": "object",
+      "properties": {
+        "defaultMetrics": {
+          "description": "Default metrics map {resourceType : list of metrics config} (Refer for metrics structure: https://docs.microsoft.com/en-us/rest/api/monitor/metric-definitions/list#metricdefinition)",
+          "type": "object",
+          "additionalProperties": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/DefaultServerMetricsConfigModel"
+            }
+          }
+        }
+      }
+    },
+    "DefaultServerMetricsConfigModel": {
+      "description": "Metrics config model",
+      "type": "object",
+      "properties": {
+        "metricnamespace": {
+          "type": "string"
+        },
+        "aggregation": {
+          "type": "string"
+        },
+        "name": {
+          "$ref": "#/definitions/LocalizedName"
+        },
+        "unit": {
+          "type": "string"
+        },
+        "displayDescription": {
+          "type": "string"
+        }
+      }
+    },
+    "Error": {
+      "description": "Error from a REST request.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message.",
+          "type": "string"
+        },
+        "target": {
+          "description": "The error target.",
+          "type": "string"
+        },
+        "details": {
+          "description": "Additional details and inner errors.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorResponseBody"
```

Oops! One more little problem.
```suggestion
            "$ref": "#/definitions/Error"
```

---

changed to error model